# Icelandic translation for apport
# Copyright (c) 2008 Rosetta Contributors and Canonical Ltd 2008
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2008.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-02-21 18:59+0100\n"
"PO-Revision-Date: 2021-10-18 22:33+0000\n"
"Last-Translator: Sveinn í Felli <sv1@fellsnet.is>\n"
"Language-Team: Icelandic <is@li.org>\n"
"Language: is\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Safna kerfisupplýsingum"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Auðkenningar er krafist fyrir söfnun á kerfisupplýsingum fyrir þessa "
"villuskýrslu"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Vandamálaskýrslur kerfis"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr "Sláðu inn lykilorð til að fá aðgang að villuskýrslum kerfisforrita"

#: ../apport/ui.py:259
msgid "This package does not seem to be installed correctly"
msgstr "Þessi pakki virðist ekki rétt upp settur"

#: ../apport/ui.py:269
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:299
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Þú er með uppsettar einhverjar úreltar útgáfur af pökkum. Endilega uppfærðu "
"eftirfarandi pakka og athugaðu hvort vandamálið endurtekur sig:\n"
"\n"
"%s"

#: ../apport/ui.py:450
msgid "unknown program"
msgstr "óþekkt forrit"

#: ../apport/ui.py:452
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Fyrirgefðu, forritið \"%s\" hætti óvænt"

#: ../apport/ui.py:454
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Tölvan þín hefur ekki nóg laust innra minni til að greina sjálfkrafa "
"vandamálið og senda til forritaranna."

#: ../apport/ui.py:459 ../apport/ui.py:1880
#, python-format
msgid "Problem in %s"
msgstr "Vandamál í %s"

#: ../apport/ui.py:515 ../apport/ui.py:532 ../apport/ui.py:698
#: ../apport/ui.py:704 ../apport/ui.py:927 ../apport/ui.py:1691
#: ../apport/ui.py:1847 ../apport/ui.py:1853
msgid "Invalid problem report"
msgstr "Ógild vandamálsskýrsla"

#: ../apport/ui.py:516
msgid "You are not allowed to access this problem report."
msgstr "Þú hefur ekki leyfi til að skoða þessa skýrslu."

#: ../apport/ui.py:524
msgid "Error"
msgstr "Villa"

#: ../apport/ui.py:526
msgid "There is not enough disk space available to process this report."
msgstr "Það er ekki nóg diskpláss til að vinna úr skýrslunni."

#: ../apport/ui.py:562
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:563
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:572 ../apport/ui.py:677
msgid "Invalid PID"
msgstr "Ógilt PID"

#: ../apport/ui.py:572
msgid "The specified process ID does not exist."
msgstr ""

#: ../apport/ui.py:577
msgid "Not your PID"
msgstr ""

#: ../apport/ui.py:578
msgid "The specified process ID does not belong to you."
msgstr ""

#: ../apport/ui.py:635
msgid "No package specified"
msgstr "Enginn pakki valinn"

#: ../apport/ui.py:637
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Þú þarft að tilgreina pakka eða PID. Sjá --help fyrir meiri upplýsingar."

#: ../apport/ui.py:664
msgid "Permission denied"
msgstr "Aðgangur bannaður"

#: ../apport/ui.py:666
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Tiltekið ferli tilheyrir þér ekki. Keyrðu þetta forrit sem eigandi ferlisins "
"eða sem kerfisstjórnandi"

#: ../apport/ui.py:678
msgid "The specified process ID does not belong to a program."
msgstr "Þetta ID tilheyrir ekki neinu forriti."

#: ../apport/ui.py:699
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr ""

#: ../apport/ui.py:705
#, python-format
msgid "Package %s does not exist"
msgstr "Pakki %s er ekki til"

#: ../apport/ui.py:734 ../apport/ui.py:932 ../apport/ui.py:965
#: ../apport/ui.py:972
msgid "Cannot create report"
msgstr "Get ekki gert skýrslu"

#: ../apport/ui.py:749 ../apport/ui.py:803 ../apport/ui.py:820
msgid "Updating problem report"
msgstr "Uppfæri villuskýrslu"

#: ../apport/ui.py:751
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Þú ert ekki tilkynnandi eða áskrifandi að þessarri villuskýrslu, eða "
"skýrslan er tvítekin eða nú þegar lokuð.\n"
"\n"
"Búðu til nýja skýrslu með ‚apport-bug‘."

#: ../apport/ui.py:763
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Þú ert ekki tilkynnandi þessarar villuskýrslu. Mun auðveldara er að merkja "
"villu sem tvítekningu annarrar heldur en að færa athugasemdir og viðhengin "
"þín í nýja villu.\n"
"\n"
"Þar af leiðir ráðleggjum við að þú sendir inn nýja villuskýrslu með ‚apport-"
"bug‘ og búir til athugasemd í þessarri villu um þá skýrslu.\n"
"\n"
"Ertu alveg viss þú viljir halda áfram?"

#: ../apport/ui.py:803 ../apport/ui.py:820
msgid "No additional information collected."
msgstr "Engum aukaupplýsingum er safnað"

#: ../apport/ui.py:876
msgid "What kind of problem do you want to report?"
msgstr "Hvers konar forrit viltu senda skýrslu um?"

#: ../apport/ui.py:895
msgid "Unknown symptom"
msgstr "Óþekkt einkenni"

#: ../apport/ui.py:896
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Einkennið „%s“ er ekki þekkt."

#: ../apport/ui.py:934
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:950
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Eftir lokun þessa skilaboða, smelltu á forritsglugga til að skrá villu um "
"hann."

#: ../apport/ui.py:966 ../apport/ui.py:973
msgid "xprop failed to determine process ID of the window"
msgstr ""

#: ../apport/ui.py:988
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:989
msgid "Specify package name."
msgstr "Tilgreindu pakkaheiti."

#: ../apport/ui.py:995 ../apport/ui.py:1125
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""

#: ../apport/ui.py:1033
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:1044
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""

#: ../apport/ui.py:1053
msgid "Click a window as a target for filing a problem report."
msgstr ""

#: ../apport/ui.py:1061
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""

#: ../apport/ui.py:1069
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:1078
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""

#: ../apport/ui.py:1089
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:1097
msgid "The provided pid is a hanging application."
msgstr "Gefna PID-ið er forrit sem hangir."

#: ../apport/ui.py:1105
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""

#: ../apport/ui.py:1115
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""

#: ../apport/ui.py:1131
msgid "Print the Apport version number."
msgstr "Gefa Apport útgáfunúmerið."

#: ../apport/ui.py:1292
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""

#: ../apport/ui.py:1296
msgid "Run gdb session"
msgstr ""

#: ../apport/ui.py:1297
msgid "Run gdb session without downloading debug symbols"
msgstr ""

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1299
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr ""

#: ../apport/ui.py:1353
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1357
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1435 ../apport/ui.py:1448
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "Þessi vandamálaskýrsla á við forrit sem er ekki uppsett lengur."

#: ../apport/ui.py:1468
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Vandamálið kom fyrir í forritunu %s sem er breytt síðan síðasta hrun varð."

#. can happen with broken core dumps
#. can happen with broken gz values
#: ../apport/ui.py:1527 ../apport/ui.py:1639 ../apport/ui.py:1851
msgid "This problem report is damaged and cannot be processed."
msgstr "Skýrslan er löskuð og ég get ekki unnið úr henni"

#: ../apport/ui.py:1533
msgid "This report is about a package that is not installed."
msgstr ""

#: ../apport/ui.py:1540
msgid "An error occurred while attempting to process this problem report:"
msgstr "Villa kom upp þegar reynt var að vinna þessa villuskýrslu:"

#: ../apport/ui.py:1557
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1562
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1563
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1601
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1606
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1692
msgid "Could not determine the package or source package name."
msgstr "Gat ekki ákveðið nafn pakka eða grunnpakka"

#: ../apport/ui.py:1716
msgid "Unable to start web browser"
msgstr "Gat ekki ræst vefvafra"

#: ../apport/ui.py:1717
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Gat ekki ræst vefvafra til að opna %s"

#: ../apport/ui.py:1809
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr "Get ekki tengst við gagnagrunn, athugaðu nettenginguna."

#: ../apport/ui.py:1812
msgid "Network problem"
msgstr "Vandamál með nettengingu"

#: ../apport/ui.py:1838
msgid "Memory exhaustion"
msgstr "Minnisvandamál"

#: ../apport/ui.py:1840
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Kerfið þitt hefur ekki nóg vinnsluminni til að vinna úr þessari skýrslu."

#: ../apport/ui.py:1885
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Ekki hægt að tilkynna vandamálið:\n"
"\n"
"%s"

#: ../apport/ui.py:1939 ../apport/ui.py:1951
msgid "Problem already known"
msgstr "Vandamál þegar þekkt"

#: ../apport/ui.py:1941
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Þetta vandamál var þegar tilkynnt í villuskýrslunni sem birtist í "
"vefvafranum. Athugaðu hvort þú getur bætt við einhverjum gagnlegum "
"upplýsingum."

#: ../apport/ui.py:1952
msgid "This problem was already reported to developers. Thank you!"
msgstr ""
"Þetta vandamál hefur nú þegar verið tilkynnt til þróunarteymis. Takk fyrir!"

#: ../bin/apport-cli.py:87
msgid "Press any key to continue..."
msgstr "Ýttu á einhvern lykil til að halda áfram..."

#: ../bin/apport-cli.py:94
msgid "What would you like to do? Your options are:"
msgstr "Hvað viltu gera? Þú getur:"

#: ../bin/apport-cli.py:107
#, python-format
msgid "Please choose (%s):"
msgstr "Veldu (%s);"

#: ../bin/apport-cli.py:167
#, python-format
msgid "(%i bytes)"
msgstr "(%i bæti)"

#: ../bin/apport-cli.py:169 ../gtk/apport-gtk.py:138 ../kde/apport-kde.py:416
msgid "(binary data)"
msgstr "(gögn í tvíundakerfi)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.ui.h:6 ../kde/apport-kde.py:208
msgid "Send problem report to the developers?"
msgstr "Senda vandamálsskýrslu til forritara?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Eftir að vandamálaskýrslan hefur verið send opnast sjálfkrafa netvafri með "
"formi.\n"
"Vinsamlegast fylltu það út."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&Senda skýrslu (%s)"

#: ../bin/apport-cli.py:222
msgid "&Examine locally"
msgstr "Skoða &hér"

#: ../bin/apport-cli.py:226
msgid "&View report"
msgstr "&Lesa skýrslu"

#: ../bin/apport-cli.py:228
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "&Geyma skýrslu til að senda inn seinna eða til að færa á annan stað"

#: ../bin/apport-cli.py:231
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Hætta við og h&unsa frekari hrun í þessari útgáfu forritsins"

#: ../bin/apport-cli.py:234 ../bin/apport-cli.py:314 ../bin/apport-cli.py:348
#: ../bin/apport-cli.py:369
msgid "&Cancel"
msgstr "&Hætta við"

#: ../bin/apport-cli.py:255
msgid "Problem report file:"
msgstr "Skrá með upplýsingar um vandamál:"

#: ../bin/apport-cli.py:261 ../bin/apport-cli.py:266
msgid "&Confirm"
msgstr "&Staðfesta"

#: ../bin/apport-cli.py:265
#, python-format
msgid "Error: %s"
msgstr "Villa: %s"

#: ../bin/apport-cli.py:271 ../kde/apport-kde.py:464
msgid "Collecting problem information"
msgstr "Safna saman upplýsingum um vandamál"

#: ../bin/apport-cli.py:273
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Þú getur sent inn skýrslu með þessum upplýsingum til að bæta forritið.\n"
"Þetta getur tekið nokkra stund."

#: ../bin/apport-cli.py:289 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:514
msgid "Uploading problem information"
msgstr "Sendi vandamálsskýrslu"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Verið er að senda upplýsingar sem safnað hefur verið í villukerfi.\n"
"Það getur tekið nokkrar mínútur."

#: ../bin/apport-cli.py:347
msgid "&Done"
msgstr "&Lokið"

#: ../bin/apport-cli.py:353
msgid "none"
msgstr "ekkert"

#: ../bin/apport-cli.py:354
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Valið: %s. Margir valkostir:"

#: ../bin/apport-cli.py:370
msgid "Choices:"
msgstr "Valkostir:"

#: ../bin/apport-cli.py:384
msgid "Path to file (Enter to cancel):"
msgstr "Slóð skráar (Enter til að hætta við):"

#: ../bin/apport-cli.py:390
msgid "File does not exist."
msgstr "Skrá er ekki til."

#: ../bin/apport-cli.py:392
msgid "This is a directory."
msgstr "Þetta er mappa."

#: ../bin/apport-cli.py:397
msgid "To continue, you must visit the following URL:"
msgstr "Þú verður að heimsækja eftirfarandi slóð til að halda áfram:"

#: ../bin/apport-cli.py:399
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Þú getur ræst vafra núna eða afritað þessa slóð í vafra á annari tölvu."

#: ../bin/apport-cli.py:404
msgid "Launch a browser now"
msgstr "Ræsa netvafra"

#: ../bin/apport-cli.py:419
msgid "No pending crash reports. Try --help for more information."
msgstr ""

#: ../bin/apport-retrace.py:48
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""

#: ../bin/apport-retrace.py:55
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""

#: ../bin/apport-retrace.py:64
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""

#: ../bin/apport-retrace.py:73
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""

#: ../bin/apport-retrace.py:76
msgid "Override report's CoreFile"
msgstr ""

#: ../bin/apport-retrace.py:79
msgid "Override report's ExecutablePath"
msgstr ""

#: ../bin/apport-retrace.py:82
msgid "Override report's ProcMaps"
msgstr ""

#: ../bin/apport-retrace.py:88
msgid "Rebuild report's Package information"
msgstr ""

#: ../bin/apport-retrace.py:95
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""

#: ../bin/apport-retrace.py:109
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""

#: ../bin/apport-retrace.py:119
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-retrace.py:125
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""

#: ../bin/apport-retrace.py:131
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:139
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""

#: ../bin/apport-retrace.py:145
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""

#: ../bin/apport-retrace.py:156 ../bin/apport-valgrind.py:99
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""

#: ../bin/apport-retrace.py:163
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""

#: ../bin/apport-retrace.py:173
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""

#: ../bin/apport-retrace.py:181
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""

#: ../bin/apport-retrace.py:188
msgid "Do not add StacktraceSource to the report."
msgstr ""

#: ../bin/apport-retrace.py:201
msgid "You cannot use -C without -S. Stopping."
msgstr "Þú getur ekki notað -C án -S. Hætti."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:231
msgid "OK to send these as attachments? [y/n]"
msgstr "Er allt í lagi að senda þetta sem sem viðhengi? [y/n]"

#: ../bin/apport-unpack.py:35
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:36
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:37
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:86
msgid "Destination directory exists and is not empty."
msgstr ""

#: ../bin/apport-valgrind.py:40
msgid "See man page for details."
msgstr "Sjá nánar á man-síðu."

#: ../bin/apport-valgrind.py:49
msgid "specify the log file name produced by valgrind"
msgstr ""

#: ../bin/apport-valgrind.py:55
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"endurnýta sandkassamöppu (SDIR) sem þegar hefur verið búin til eða, ef hún "
"er ekki til, búa hana til"

#: ../bin/apport-valgrind.py:63
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"ekki búa til né endurnýta sandkassamöppu fyrir auka aflúsunarmerki heldur "
"aðeins treysta á uppsett aflúsunarmerki."

#: ../bin/apport-valgrind.py:72
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"endurnýta skyndiminnismöppu (CDIR) sem þegar hefur verið búin til eða, ef "
"hún er ekki til, búa hana til"

#: ../bin/apport-valgrind.py:81
msgid "report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-valgrind.py:88
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:130
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Villa: %s er ekki keyranlegt forrit. Hætti."

#: ../data/apportcheckresume.py:75
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:80
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:88
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Endurvakningin hékk í blá endann en lítur út fyrir að hafa klárast eðlilega."

#: ../data/kernel_oops.py:33
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Kerfið þitt verður kannski óstöðugt núna og þarf hugsanlega að endurræsa það."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Tilkynna vandamál..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Tilkynna bilun til forritara"

#: ../gtk/apport-gtk.py:155
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Því miður, forritið %s hætti óvænt."

#: ../gtk/apport-gtk.py:158
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Því miður, forritið %s lokaði óvænt."

#: ../gtk/apport-gtk.py:163 ../kde/apport-kde.py:223 ../kde/apport-kde.py:272
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Því miður, %s hefur lent í innri villu."

#: ../gtk/apport-gtk.py:186 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:216
msgid "Send"
msgstr "Senda"

#: ../gtk/apport-gtk.py:207 ../gtk/apport-gtk.py:613 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:333
msgid "Show Details"
msgstr "Sýna ítarlegri upplýsingar"

#: ../gtk/apport-gtk.py:220 ../kde/apport-kde.py:264 ../kde/apport-kde.py:279
msgid "Continue"
msgstr "Halda áfram"

#: ../gtk/apport-gtk.py:244
#, python-format
msgid "The application %s has stopped responding."
msgstr "Forritið %s hætti að svara."

#: ../gtk/apport-gtk.py:248
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Forritið %s hætti að svara."

#: ../gtk/apport-gtk.py:265 ../kde/apport-kde.py:231
#, python-format
msgid "Package: %s"
msgstr "Pakki: %s"

#: ../gtk/apport-gtk.py:271 ../kde/apport-kde.py:238
msgid "Sorry, a problem occurred while installing software."
msgstr "Því miður, vandamál kom upp við upppsetningu hugbúnaðar."

#: ../gtk/apport-gtk.py:281 ../gtk/apport-gtk.py:300 ../kde/apport-kde.py:246
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "Forritið %s lenti í innri villu."

#: ../gtk/apport-gtk.py:283 ../kde/apport-kde.py:251
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Forritið %s lokaði óvænt."

#: ../gtk/apport-gtk.py:308 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:276
msgid "If you notice further problems, try restarting the computer."
msgstr "Ef þú verður var við fleiri vandamál, reyndu þá að endurræsa tölvuna."

#: ../gtk/apport-gtk.py:311 ../kde/apport-kde.py:281
msgid "Ignore future problems of this type"
msgstr "Hunsa svona vandamál héðan í frá"

#: ../gtk/apport-gtk.py:617 ../kde/apport-kde.py:330
msgid "Hide Details"
msgstr "Fela nánari upplýsingar"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:383 ../kde/apport-kde.py:553
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Hætta við"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "Í lagi"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Hrunskýrsla"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Því miður, upp kom innri villa.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr ""

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Hunsa frekari vandamál í þessarri útgáfu forrits"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr ""

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "Skoða _hér"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr ""

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Safna upplýsingum um vandamál</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Er að safna upplýsingum sem gætu hjálpað forriturum að laga vandann sem þú "
"tilkynnir"

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Sendi vandamálsskýrslu</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:516
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Er að senda upplýsingarnar á villukerfið. Þetta gæti tekið nokkrar mínútur."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport villuskrá"

#: ../kde/apport-kde.py:267
msgid "Leave Closed"
msgstr "Skilja eftir lokað"

#: ../kde/apport-kde.py:268 ../kde/apport-kde.py:434
msgid "Relaunch"
msgstr "Keyra aftur upp"

#: ../kde/apport-kde.py:360
msgid "Username:"
msgstr "Notandanafn:"

#: ../kde/apport-kde.py:363
msgid "Password:"
msgstr "Lykilorð:"

#: ../kde/apport-kde.py:463
msgid "Collecting Problem Information"
msgstr "Safna saman upplýsingum um vandamál"

#: ../kde/apport-kde.py:466
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Söfnuðum upplýsingum er hægt að koma til forritaranna til að bæta forritið. "
"Þetta gæti tekið nokkrar mínútur."

#: ../kde/apport-kde.py:513
msgid "Uploading Problem Information"
msgstr "Sendi upplýsingar um vandamálið"
